### ISSUE TITLE
Only HVAC Mode is the initial set

### ISSUE THREAD

**Issue by oxymoron1629** [swingerman/ha-dual-smart-thermostat#5]
Hi, loving this integration, finally met a need that I was trying to fix. Is it possible to select which HVAC mode the climate integration is on via the Lovelace? Using the developer tools, I can manually set the HVAC to any mode (off, cool, heat, heat_cool) but that option isn't available in Lovelace. See image below.

![image](https://user-images.githubusercontent.com/69619724/157168930-33f10d35-89d1-4197-bfea-476198d8dbf1.png)

Is this something in the works? Is this a known bug I missed? Thanks in advance.

**Comment by swingerman** [comment]
Thanks for finding this. I think It's possible to enable all HVAC modes that are possible based on the connected sensors. I will look into this as soon as possible.

**Comment by ejohb** [comment]
Working well for me generally but this is a problem.

**Comment by kgalford1** [comment]
`v0.6.0`

All HVAC modes are appearing now, but when Heat/Cool is selected, both low and high handles appear, whereas I am expecting just one handle for each.

Whenever the a handle is changed, the other reverts to `target_temp_low/high`, and vice-versa. In addition whenever the current temperature changes, the target values (but not the visual handles) seem to reset to `target_temp_low/high`.

![image](https://user-images.githubusercontent.com/6301097/168422107-63b779d9-b744-4026-b1bf-c0dacd4be16e.png)

If people are unable to recreate this let me know and I'll record a video.

```
climate:
  - platform: dual_smart_thermostat
    name: Den
    heater: switch.thermy_heater
    cooler: switch.thermy_cooler
    target_sensor: sensor.apparent_temperature
    min_temp: 10
    max_temp: 30
    ac_mode: false
    target_temp: 20
    target_temp_high: 24
    target_temp_low: 16
    cold_tolerance: 0.3
    hot_tolerance: 0
    min_cycle_duration:
      seconds: 5
    keep_alive:
      minutes: 3
    away_temp: 16
    precision: 0.1
```

**Comment by kgalford1** [comment]
I've got the dev env up and running. I'll create a proper issue for my comment above soon and should be able to implement the fix.

**Comment by swingerman** [comment]
I partially fixed this in v0.6.1 but it's still buggy and unusable. I will completely fix it in the next patch
